modules/programs/alacritty.nix

```diff
@@ -40,8 +56,31 @@ in {
   };
 
   config = lib.mkIf cfg.enable {
+    assertions = [{
+      # If using the theme option, ensure that theme exists in the
+      # alacritty-theme package.
+      assertion = let
+        available = lib.pipe "${pkgs.alacritty-theme}/share/alacritty-theme" [
+          builtins.readDir
+          (lib.filterAttrs
+            (name: type: type == "regular" && lib.hasSuffix ".toml" name))
+          lib.attrNames
+          (lib.map (lib.removeSuffix ".toml"))
+        ];
+      in cfg.theme == null || (builtins.elem cfg.theme available);
```

Comment:
This will fail to eval for users that have disabled IFD.

Reply:
IFD essentially happens whenever you have Nix code which depends on the content of a package (it's not about actually `import`ing, any logic which depends on the contents of a derivation to keep evaluating is called IFD).

Reply:
Well, I guess we can just remove the assertion, then. It kinda sucks that we can't assert basic things like this, but if it's breaking things, then I guess there's no way.